dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -1084,6 +1105,7 @@ export const decidePalette = (format: ArticleFormat): Palette => {
 			headlineByline: textHeadlineByline(format),
 			standfirst: textStandfirst(format),
 			standfirstLink: textStandfirstLink(format),
+			lastUpdatedStandfirst: textLastUpdatedStandfirst(format),
```

Comment:
Is there a better name for this? Having 'standfirst' here is confusing as I don't _think_ this is related to the standfirst text?

Reply:
Do you think just `lastUpdated` would be good? or maybe `articleLastUpdated`?

Reply:
👍 from me